.NET/Microsoft.Recognizers.Text.Number/Models/NumberModel.cs

```diff
@@ -7,6 +7,11 @@ public NumberModel(IParser parser, IExtractor extractor)
         {
         }
 
+        public NumberModel(IParser parser, IExtractor extractor, bool recode)
```

Comment:
Please do not include new public methods in the public consumer-level interface. This can be considered a breaking change as users won't use it without special knowledge.